src/screens/dialogs/access-file-dialog.tsx

```diff
@@ -35,9 +53,7 @@ export const AccessManagementDialog = () => {
     if (Open && selectedFile) {
       fetchUsersWithAccess();
     }
-  }, [selectedFile]);
-
-  const handleSave = async () => {};
+  }, [selectedFile, shared]);
```

Comment:
This works when you share with one user, but try sharing with a second user, the list is not updated. You can get rid of these dependencies if you do what I said in previous comments.